doc/getting_started.mld

```diff
@@ -75,6 +75,8 @@ Options can be modified by the means of:
 [.ocamlformat] files in the containing and all ancestor directories for each input file are used, as well as the global [.ocamlformat] file defined in [$XDG_CONFIG_HOME/ocamlformat]. The global [.ocamlformat] file has the lowest priority, then the closer the directory is to the processed file, the higher the priority.
 
 When the option [--enable-outside-detected-project] is set, [.ocamlformat] files outside of the project (including the one in [XDG_CONFIG_HOME]) are read. The project root of an input file is taken to be the nearest ancestor directory that contains a [.git] or [.hg] or [dune-project] file.
+For more details, read about {{!page-configuration}how OCamlFormat finds its root project and computes its configuration}.
```

Comment:
I think this sentence should entirely replace the previous three paragraphs. Users only need to know about `.ocamlformat` at the root to get started.